Week2/homework/maartjes_work.js

```diff
@@ -44,4 +44,12 @@ const tuesday = [
 
 const tasks = monday.concat(tuesday);
 
-// Add your code here
+
+const salary = tasks.map(x => x.duration / 60)
```

Comment:
you could consider sending arguments towards the function salary, something like this:

const calculateEarnings = (arrayOfTasks, hourlyWage) =>
  arrayOfTasks.map((task) => task.duration / 60)
  .filter((hoursAmount) => hoursAmount >= 2)
  .map((paidHoursAmount) => paidHoursAmount * hourlyWage)
  .reduce((sum, amount) => sum + amount, 0)
  .toFixed(2);

console.log('Maartje has earned € ' + calculateEarnings(tasks, 16));